study-builder/studies/prion/emails/contact_medical_reminder.en.html

```diff
@@ -0,0 +1,83 @@
+<html>
+<head>
+    <title></title>
+</head>
+<body>
+<div>&nbsp;
+    <div dir="ltr" style="margin-left: 0;">
+        <table style="border: none; border-collapse: collapse;">
+            <colgroup>
+                <col width="593"/>
+            </colgroup>
+            <tbody>
+            <tr style="height: 0">
+                <td style="border: 0 solid rgb(0, 0, 0); vertical-align: top; padding: 31px 0 0;">
+                    <img src="https://storage.googleapis.com/dev-prion-assets/images/project-logo-dark.png"
```

Comment:
I recommend taking advantage of substitutions so we don't hardcode to a DEV bucket.

Reply:
The tricky thing here is that this isn't actually using any Pepper features--it's going to be sent manually with SendGrid.  So I'm not totally sure if there is a great way to do substitutions, although I could at least do the thing I did below.